Packs/ReblazeWAF/ParsingRules/ReblazeWAF/ReblazeWAF.xif

```diff
@@ -0,0 +1,32 @@
+[RULE:Strauss_logs]
```

Comment:
For the Parsing Rule, you need another naming convention- Customer name cannot exist in the rule.

Also, the same comment in the Modeling Rule- The field created by the Parsing Rule should be, "parsed_fields".
This is the convention we use, please check it with other packs.